cli_tools/gce_vm_image_import/importer/api_inflater.go

```diff
@@ -73,6 +77,13 @@ func (inflater *apiInflater) addTraceLog(l string) {
 }
 
 func (inflater *apiInflater) inflate(loggableBuilder *service.SingleImageImportLoggableBuilder) (persistentDisk, inflationInfo, error) {
+	// Block cancel()
+	inflater.inflatingSem <- "inflating"
```

Comment:
Who consumes this?

Reply:
It behaves as a semaphore. As the comment line, it blocks cancel(). The content of string is not useful, just a kind of explanation.